bundles/net.tourbook/src/net/tourbook/extension/export/ActionExport.java

```diff
@@ -155,6 +156,7 @@ private ArrayList<ExportTourExtension> getExtensionPoints() {
       }
 
       for (final IExtension extension : extPoint.getExtensions()) {
+
```

Comment:
I would sort the export extionsions alphabetically except `.mt`, this should be at the end with a separator

Reply:
> I would sort the export extensions alphabetically except `.mt`, this should be at the end with a separator

I can sort them.
What kind of separator ? a new line ?

Reply:
Like the other separators

![image](https://user-images.githubusercontent.com/1283445/236600605-86d06c5f-1ce6-4329-87d8-feab533849ee.png)

Reply:
Fixed

![image](https://user-images.githubusercontent.com/25235665/236922651-f18bc044-5085-481d-92ba-82b3dfbe12c5.png)

Reply:
Looks now much better 👍

Reply:
I replaced the MT icon with the dark border

![image](https://user-images.githubusercontent.com/1283445/237005177-86dae17b-3c8d-4b2c-ac40-bb8bc93f9760.png)

Reply:
> I replaced the MT icon with the dark border in [22a486c](https://github.com/mytourbook/mytourbook/commit/22a486cbd82545b6c793d4d9bd0339bb85692bc5)
> 
Good, I hadn't noticed